### PR TITLE
Add Appendix D

### PR DESCRIPTION
**Pull request by juxd**
Add use cases to developer guide.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -895,35 +895,122 @@ what specific customers are ordering
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is FoodZoom and the *Actor* is the `user`, unless specified otherwise)
```
Comment (kohchihao) [inline docs/DeveloperGuide.adoc]: Shouldn't the Actor be `Manager` and `Delivery man` ?

Comment (juxd) [reply docs/DeveloperGuide.adoc]: makes sense... i was thinking more of the manager/deliveryman as a user

Comment (kohchihao) [reply docs/DeveloperGuide.adoc]: hmm. I think since we used `Manager` and `Delivery man` inside our use cases, should be consistent.

Comment (juxd) [reply docs/DeveloperGuide.adoc]: ok, will change.

**Pull request merged**